workers/reporter/docker/Reporter.Dockerfile

```diff
@@ -45,6 +45,9 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
     apt-get update \
```

Comment:
Commit message nit: instead of _commands_, maybe rather _programs_ or _executables_ or even _clients_ ?

Reply:
I would prefer to use `packages` to stay consistent with the apt-get [documentation](https://linux.die.net/man/8/apt-get).